### PR TITLE
add some annotations for ambient traffic redirection 

### PR DESCRIPTION
**Pull request by zengyuxing007**
ref: https://github.com/istio/istio/issues/49829

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]

**Review by howardjohn — COMMENTED** [review]
These LGTM as documentation in general, but not for ambient necessarily; that is seperate

**Review by kfaseela — APPROVED** [review]

File: annotation/annotations.yaml [hunk 1]
```diff
@@ -530,3 +530,66 @@ annotations:
     hidden: true
     resources:
       - Any
+
+  - name: traffic.istio.io/includeOutboundIPRanges
```
Comment (zirain) [inline annotation/annotations.yaml:534]: Need to highlight these annotations are used for ambient

**Pull request merged**